src/mds/Server.cc

```diff
@@ -1242,6 +1251,15 @@ void Server::evict_cap_revoke_non_responders() {
   }
 
   auto&& to_evict = mds->locker->get_late_revoking_clients(cap_revoke_eviction_timeout);
+  // don't kick client if ops or osds are laggy
+  if(to_evict.size() && mds->objecter->with_osdmap([](const OSDMap &map)
+  { return map.are_osds_laggy(); })) {
```

Comment:
```suggestion
  bool laggy = mds->objecter->with_osdmap([](auto &map) { return map.are_osds_laggy(); }
  if(to_evict.size() && laggy)) {
```

Is more visually clear. Embedding a lambda in an if condition is hard to read.

Reply:
yeah, i made the change. thanks.